onnxruntime/core/framework/sequential_execution_plan.h

```diff
@@ -103,7 +101,7 @@ struct SequentialExecutionPlan : public ExecutionPlanBase {
   class ExecutionStep {
    public:
     virtual ~ExecutionStep() {}
-    virtual StepCommandFn GetStepFun() = 0;
+    virtual Status Execute(void* ctx, size_t stream_idx, bool& continue_flag) = 0;
```

Comment:
not related with this PR: we probably want to replace the void* here as the ExecutionContext*. 
The void* is the hack i put there when prototype.